### ISSUE TITLE
Build a script/tool to generate updates to catalog templates and also to sanity check them

### ISSUE THREAD

**Issue by leodotcloud** [rancher/rancher#9567]
When a new version of a micro service is release, a new catalog template has to be generated. This can involve creating a new folder or using an existing folder, involves fields like `upgrade_from`, `minimum_rancher_version`, `maximum_rancher_version`, etc.

Having a script inside the repo helps the developer to generate these changes automatically by answering few questions and also helps to sanity test the changes.

In short, take the intelligence from @deniseschannon's head and build a script/tool :-)

**Comment by loganhz** [comment]
Thanks for your report!

With the release of Rancher 2.0, development on v1.6 is only limited to critical bug fixes and security patches.

If you think we should keep this issue open, please let me know.
